frontend/src/pages/files/index.tsx

```diff
@@ -100,7 +102,15 @@ const Files = () => {
           type={AlertMessage.ERRROR}
           title="Error"
           message={error}
-          onClose={handleSetError}
+          onClose={handleAlertClose}
+        />
+      )}
+      {success && (
+        <AlertPortal
+          type={AlertMessage.SUCCESS}
```

Comment:
why use portal twice just to pass different props? this is kind of antipattern?

Reply:
I agree, it will be changed